plugins/node.d.openbsd/cpu

```diff
@@ -49,8 +49,9 @@ if [ "$1" = "config" ]; then
 #	SYSCRITICAL=$PERCENT*50/100
 #	INTWARNING=$PERCENT*80/100
 #	USRWARNING=$PERCENT*80/100
+#	SPINWARNING=$PERCENT*80/100
 	echo 'graph_title CPU usage'
-	echo 'graph_order system interrupt user nice idle'
+	echo 'graph_order system spin interrupt user nice idle'
```

Comment:
I think it's logical to keep the idle value at the end of the graph as it fill the whole graph to the top.
About the color change, as the color scheme is automatic and that spin shouldn't be added after idle, how could it be managed?

Reply:
Both arguments (*keeping colors unchanged* vs. *keeping idle at the end*) sound reasonable.
How about adding the new value as the second-to-last item?

Regarding specifying colors: it is possible, but surely too much complication here. Simple plugins are good plugins :)

Reply:
@steveschnepp: what is your opinion towards "colors should not change" vs. "idle should be the bottom of the graph"?

Reply:
> Regarding specifying colors: it is possible, but surely too much complication here
Well, I'd argue that in this particular case colors should be specified.

Reply:
Maybe let us bring this discussion to an end ...

Personally, I am very reluctant to add manual color handling to a plugin. I would do it, in case of a really counter-intuitive color assignment (e.g. "red" for "idle"), but I guess, this is not the case here.

@steveschnepp: if you feel strongly about this, then maybe just merge the commit and add the color scheme. Otherwise I would merge it and move the new field to the second-to-last position. Both should be acceptable for the users - and they will enjoy the new field to be shown in their graphs ...